templates/webhook.bin.erb

```diff
@@ -250,6 +259,24 @@ class Server < Sinatra::Base
       message
     end
 
+    def generate_types(environment)
+      begin
+        command = "#{$command_prefix} /opt/puppetlabs/bin/puppet generate types --environment #{environment}"
```

Comment:
In my environments, I get a handful of errors, but `generate types` still returns `0`.
eg
```
[root@puppet4server-ci01c ~]# umask 0022;  /opt/puppetlabs/bin/puppet generate types --environment GIT_dev_generate_types
Notice: Generating Puppet resource types.
Error: /etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/openldap/lib/puppet/type/openldap_access.rb: title patterns that use procs are not supported.
Error: Could not autoload puppet/provider/openldap_database/augeas: undefined method `provider' for nil:NilClass
Error: Failed to load custom type 'openldap_database' from '/etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/openldap/lib/puppet/type/openldap_database.rb': Could not autoload puppet/provider/openldap_database/augeas: undefined method `provider' for nil:NilClass
Error: /etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/openldap/lib/puppet/type/openldap_dbindex.rb: title patterns that use procs are not supported.
Error: /etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/openldap/lib/puppet/type/openldap_overlay.rb: title patterns that use procs are not supported.
Error: /etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/wildfly/lib/puppet/type/wildfly_resource.rb: title patterns that use procs are not supported.
Error: /etc/puppetlabs/code/environments/GIT_dev_generate_types/modules/yamlfile/lib/puppet/type/yaml_setting.rb: title patterns that use procs are not supported.
[root@puppet4server-ci01c ~]# echo $?
0
```